packages/engine/tests/units/message/remove_self/src/init.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "agent_name": "to-be-removed",
```

Comment:
Very small thing but as I half mentioned before, could we swap the order of these agents? I'm just wary of default or first values

Reply:
Good point, I added another agent, so we have three agent with the names `"1", "to-be-removed", "3".